products/health-care/appointments/va-online-scheduling/research/may-2021-facilities-personalization-research/conversation-guide.md

```diff
@@ -82,9 +82,9 @@ So where do you currently receive health care, VA or otherwise?
 
 -   How many different VA locations have you gone to for care?
 
-    -   If more than one: why do you go to some of the other locations?
+    -   If more than one: why do you go to some of the other locations? ```comment: This question works well if they are going to multiple facilities currently, but if they say that they have gone to more than one location b/c they moved or something in the question before this, this question doesn't make sense. However, it might be worth asking them directly in a separate question if they currently go to more than one VA facility for care. Be preapred - You may get answers like CBOCs and main facilities, which isn't quite what you're looking for. ```
```

Comment:
I think it's okay if we get CBOCs or main facilities, we'll just have to keep an eye out for that. Keeping it broad will help explore their preferences across all their options. Edit makes sense, something like:
- Have you gone to more than one VA location for care?
- IF YES - Why do you go to some of the other locations?